__tests__/app.test.js

```diff
@@ -28,3 +38,27 @@ describe("/api/categories", () => {
       });
   });
 });
+
+describe("/api/reviews", () => {
+  test("GET:200 responds with an array of review objects added comment_count", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then((res) => {
+        expect(res.body.reviews).toBeInstanceOf(Array);
+        res.body.reviews.forEach((review) => {
+          expect(review).toMatchObject({
+            owner: expect.any(String),
+            title: expect.any(String),
+            review_id: expect.any(Number),
+            category: expect.any(String),
+            review_img_url: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            designer: expect.any(String),
+            comment_count: expect.any(Number),
+          });
+        });
+      });
+  });
+});
```

Comment:
Just like in the categories one, because we are doing this with a loop on an array, we need to do a length check to ensure that res.body.reviews is not just an empty array but this looks great otherwise!